cesql/spec.md

```diff
@@ -307,18 +337,23 @@ Because of implicit casting, no functions with the same name and same arity but
 
 An overload on a variadic function is allowed only if the number of initial fixed arguments is greater than the maximum arity for that particular function name. Only one variadic overload is allowed.
```

Comment:
I think the wording here might need tweaking:
- "An overload on a variadic function"... reads to me as "You can create an overload of a function that's already defined with variadic args", which I don't think is the intent
- "Only one variadic overload is allowed" need to be scoped to a particular function name

Perhaps something like:
A function name MAY have at most one variadic overload definition and only if the number of initial fixed arguments is greater than the maximum arity of all other function definitions for that function name.